### DISCUSSION TITLE
Non-English Labels Sub-group

### DISCUSSION THREAD

**Discussion by Gene-Weaver** [Gene-Weaver/VoucherVision#14] [General Info & Subgroups]
This is the workspace for the Non-English label subgroup.

**Comment by mickley** [comment]
I've played around with these a bit: both with languages that use latin characters, and Russian, Chinese, and Japanese that use other character sets. It seems that Google Vision Handwriting does not recognize and OCR non-Latin characters well, at least they are not returning in the OCR text. 

Google Vision does have [support for a wide variety of languages](https://cloud.google.com/vision/docs/languages), though. Do we need to include a language hint?

**Comment by Gene-Weaver** [comment]
I have found the same. Fortunately, the Qwen-2-VL-AWQ model seems to be able to actually transcribe the other languages as written. Still haven't dug very deep beyond that though.
